lib/websocket.py

```diff
@@ -531,9 +533,9 @@ def _validate_header(self, headers, key):
             return False
         result = result.lower()
 
-        value = key + "258EAFA5-E914-47DA-95CA-C5AB0DC85B11"
+        value = key + ensure_binary("258EAFA5-E914-47DA-95CA-C5AB0DC85B11")
```

Comment:
Wouldn't a `b` prefix be more appropriate in this case?

EDIT: Same goes for all other static strings.

Reply:
Because key in python2 is str and in python3 it is a basestring. Concatenation will fail in both python2 and python3. I decided to use `ensure_binary` everywhere because I think it will be easy to track and replace all entries to b' once we ditch python2 for good